sdk/storage/tests.yml

```diff
@@ -26,6 +26,17 @@ stages:
         MatrixReplace:
           # Use dedicated storage pool in canadacentral with higher memory capacity
           - Pool=(.*)-general/$1-storage
+        MatrixConfigs:
+          - ${{ if contains(variables['Build.DefinitionName'], 'tests-weekly') }}:
+            - Name: Storage_all_versions_live_test
+              Path: sdk/storage/platform-matrix-all-versions.json
+              Selection: sparse
+              GenerateVMJobs: true
+          - ${{ if not(contains(variables['Build.DefinitionName'], 'tests-weekly')) }}:
+            - Name: Storage_live_test
+              Path: sdk/storage/platform-matrix.json
```

Comment:
Since storage uses the default matrix for python unlike java, I would delete `MatrixConfigs` and add the weekly config this way:

```
      ${{ if contains(variables['Build.DefinitionName'], 'tests-weekly') }}:
        AdditionalMatrixConfigs:
            - Name: Storage_all_versions_live_test
              Path: sdk/storage/platform-matrix-all-versions.json
              Selection: sparse
              GenerateVMJobs: true
```